sqlite-android/src/main/jni/sqlite/Android.mk

```diff
@@ -26,7 +26,13 @@ sqlite_flags := \
 	-DSQLITE_ENABLE_RTREE=1 \
 	-DSQLITE_OMIT_BUILTIN_TEST \
 	-DSQLITE_OMIT_COMPILEOPTION_DIAGS \
-	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
+	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600 \
+	-DSQLITE_DQS=0 \
+    -DSQLITE_DEFAULT_MEMSTATUS=0 \
+    -DSQLITE_MAX_EXPR_DEPTH=0 \
+    -DSQLITE_USE_ALLOCA \
+    -DSQLITE_ENABLE_BATCH_ATOMIC_WRITE \
```

Comment:
This one does not appear to be standard, however the doc says that they might just enable it by default upstream, and it being non-standard is owing more to the fact not many filesystems are F2FS (but it doesn't seem to hurt if that's the case). And specifically for Android, most devices are F2FS now I think https://en.wikipedia.org/wiki/F2FS#Adoption so this also seems like a nice optimization